### ISSUE TITLE
Proposal: Improve or disable codecov

### ISSUE THREAD

**Issue by tk0miya** [sphinx-doc/sphinx#4738]
### Problem
- qWe started to use codecov to know test coverages since a3f9935d958197a3c0963cec97755db1ff32a3a3
- But it has been failed sometimes ...
- I'd failed to configure it.

#### Procedure to reproduce the problem
See `Pull Requests` page

#### Expected results
- Configure it working well
- Or disable it

@stephenfin what do you think?

**Comment by stephenfin** [comment]
The configuration information for codecov can be found [here](https://github.com/codecov/support/wiki/Codecov-Yaml).

Based on this, it seems we probably need to configure the `status: -> project: -> target:` attribute. This defaults to `auto` which apparently means:

> [coverage] must increase from parent commit or pull request base

I'm guessing we need to start enforcing this or else disable it. Similarly, we probably need to configure the `status: -> patch: -> target:` attribute. This defaults to `80%` and our coverage is not yet that high.

If you want to focus on improving coverage, I suggest disabling the latter (patch status) for now. If you do not want to do this, you should probably disable the other one too. You can do this like so:

```
status:
  project:
    default:
      enabled: no
  patch:
    default:
      enabled: no
```

**Comment by tk0miya** [comment]
>If you want to focus on improving coverage, I suggest disabling the latter (patch status) for now. If you do not want to do this, you should probably disable the other one too. You can do this like so:

I'm not interested in coverage. So I don't have comment especially. What I only want is that the coverage does not make contributors confused.
The codecov.io was introduced from you (see #4337). So I'd like to hear your opinion.

**Comment by stephenfin** [comment]
In that case, I would disable both status checks. I can submit a PR to do this shortly.

**Comment by tk0miya** [comment]
Yes please! :-)

**Comment by stephenfin** [comment]
I've pushed a PR now. Could I also ask you to take a look at #4700 so I can move forward with my main changes, please and thank you?

**Comment by tk0miya** [comment]
Closed by #4755.
